### PR TITLE
Rename from ROSConDemo to RobotHarvestingSample Phase 1

### PR DESCRIPTION
**Pull request by spham-amzn**
Signed-off-by: Steve Pham <82231385+spham-amzn@users.noreply.github.com>

### CODE REVIEW

**Review by michalpelka — COMMENTED** [review]
I found more usages in:
- LICENSE-CC-BY-NC-4.0.txt
- docker/Dockerfile

File: Project/Gem/CMakeLists.txt [hunk 1]
```diff
@@ -22,15 +22,15 @@ if(NOT PAL_TRAIT_ROSCONDEMO_SUPPORTED)
     return()
```
[the remark this answers is not in the document]
Comment (spham-amzn) [reply Project/Gem/CMakeLists.txt:22]: Done

**Comment by adamdbrw** [comment]
@spham-amzn what is the state of this PR? Would you like us to review it?

**Comment by bhanuja-s** [comment]
Hi @spham-amzn, are there any updates to this PR? I'm updating the user guide and wondered if I should change the nomenclature within.

**Comment by spham-amzn** [comment]
> Hi @spham-amzn, are there any updates to this PR? I'm updating the user guide and wondered if I should change the nomenclature within.

I will probably close this PR since it will require a lot of updates outside of just the names (ie changing the repo name, documentation, etc)

**Pull request closed without merging**